client/app/controllers/login-error.js

```diff
@@ -0,0 +1,11 @@
+import Controller from '@ember/controller';
+
+export default class LoginErrorController extends Controller {
+  /**
+    * @return      {bool} true if login route yields error response
+    *                     about Contact not existing in CRM.
+    */
+  get contactNotAssigned() {
+    return this.model.errors.some((error) => error.response === 'CRM user not found. Please make sure your e-mail or ID is associated with an assignment.');
```

Comment:
Hmmm.. I think this could get unwieldy over time. What if the language of the original response error changes? Maybe the backend needs to provide error codes (CONTACT_NOT_ASSIGNED) that could be used for this sort of logic.

Also, AFAICT, I have never seen this emit more than one error before... I'm not sure which circumstances allow that. The JSON:API spec requires an error document to allow multiple though.

Reply:
Oh yeah, I was wary of the same. I think error codes would be fantastic! 
We've been tasked to provide more user friendly error messages, and unfortunately this is the only way I've found detect if it is specifically a CRM missing contact error.

Should we make a request of this to the CRM team? Hoping they'll have ability to modify CRM error codes.

If they can, they could even just change the CRM `error.response` value to " You are not currently assigned to a DCP project.....", so that we don't have to write special handling in our frontend code.

Reply:
No, I mean that our API would provide our own error codes — for example, where the text for this error is emitted, it might return a separate property as part of the response:
```
{
  message: "error message"
  code: "CONTACT_NOT_ASSIGNED"
}
```

These are not errors that come from CRM anyway. They come from our own app. CRM's error just says "Record not found" or something.

Reply:
Gotcha -- I shoulda caught this earlier! I have a good idea of how to go about this now.